.travis.yml

```diff
@@ -15,11 +15,9 @@ jobs:
         - sudo mv docker-compose /usr/local/bin
         - sudo service mysql stop
       install:
-        - pip install --upgrade pip MySQL-python
         - ./codalab_service.py build services -v ${TRAVIS_BRANCH} --push
-        - ./setup.sh server
       script:
-        - ./codalab_service.py start -t --mysql-port 3306 --version ${TRAVIS_BRANCH}
+        - ./codalab_service.py start -s default test --mysql-port 3306 --version ${TRAVIS_BRANCH}
```

Comment:
What's the reason for removing the -t command?

Reply:
I made all the tests run in docker for consistency (since `venv` doesn't exist anymore), and it was simpler to make `test` another "service".